lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -46,20 +47,21 @@ local function playerIsInBuild( ply )
     return !getPlayerPvpMode( ply )
 end
 
-local function movementMultiplier( weaponCount )
-    if weaponCount < 1 then return 1 end
-    return math.Clamp(1 - (1.9^( weaponCount))/100, 0, 1)
+local function movementMultiplier( totalWeight )
```

Comment:
Unless it's actually a function of how heavy each weapon is?

Reply:
it could be how heavy if  we ever have different weights for different weapons. another alternative is plyWeight.

Reply:
I would like to see customized weapon weights. Default to a certain value, but customizable nonetheless

Reply:
so totalWeight, weaponCount or plyWeight?

Reply:
The parameter should be contextual to the function. I don't care so much about consistency in this instance.

The parameter just needs to make sense, really.

Reply:
what if instead of a not affected weapons table we have a weaponWeights table and weapons not in the table default to a weight of 1. the parameter really would be a players weight then, different weapons could also have different weight.

Reply:
I like that idea!